app/controllers/github_webhooks_controller.rb

```diff
@@ -0,0 +1,36 @@
+class GithubWebhooksController < ApplicationController
+  
+  # POST /author_webhook
+  def author_webhook
+    action = params["github_webhook"]["action"]
+    case action
+    when "opened"
+      create_author(params["issue"])
+    when "edited"
+      update_author(params["issue"])
+    when "deleted"
+      delete_author(params["issue"])
+    end
+  end
+
+  def create_author(issue)
+    author = Author.create(name: issue["title"], biography: issue["body"], github_id: issue["number"])
+    author.books.create(title: Faker::Book.title, publisher: author)
```

Comment:
These two database writes should be wrapped in a transaction. As it stands you could wind up creating an author without any books if the `books.create` DB call failed for some reason.

Reply:
Sure, I added the **Transaction** block here